.gitignore

```diff
@@ -33,3 +33,5 @@ profile-1.0.0.tar.gz
 terraform.tfstate*
 terraform.tfstate.backup
 inspec-azure.plan
+
+/.vscode/
```

Comment:
Could you remove this? We'd like to avoid cluttering the .gitignore with globs for all our favorite editors. You can force git to ignore it with a global gitignore via git config --global core.excludesfile ~/.gitignore_global